modules/repository/init.go

```diff
@@ -55,6 +55,7 @@ func prepareRepoCommit(ctx models.DBContext, repo *models.Repository, tmpDir, re
 		"Description":    repo.Description,
 		"CloneURL.SSH":   cloneLink.SSH,
 		"CloneURL.HTTPS": cloneLink.HTTPS,
+		"OwnerName":	  repo.OwnerName
```

Comment:
```suggestion
		"OwnerName":	  repo.OwnerName,
```

Reply:
The above suggestion needs to be applied either in the web UI or locally and then pushed, otherwise Go cannot build it.